plotsquared/plot-membership-tiers.md

```diff
@@ -0,0 +1,48 @@
+# Plot membership tiers
+
+## Introduction
+
+PlotSquared works with the following tiers of memberships to define the different behaviors / permissions / restrictions for minecraft events, plot flags, commands etc.
+
+The member ties are one of the main part of PlotSquared by self. It's not usable for LuckPerms, but other plugins can check the membership tier for a player for a specific plot with the [PlotSquared API](/plotsquared/api/api-documentation). No configurations
```

Comment:
Clicking on the hyperlink should link within the document on gitbook. Currently, it links to GitHub.